core/trino-spi/src/main/java/module-info.java

```diff
@@ -34,6 +34,7 @@
     exports io.trino.spi.resourcegroups;
     exports io.trino.spi.security;
     exports io.trino.spi.session;
+    exports io.trino.spi.spool;
```

Comment:
should be part of previous commit, right?

Reply:
No. This is a seperate change.

Reply:
Not really - you should have protocol here before. And rename should influence that. 
Probably just a discussion should you fix a bug before or after rename...